WSCoachMarksView.m

```diff
@@ -74,14 +74,24 @@ - (void)setup {
     self.cutoutRadius = kCutoutRadius;
     self.maxLblWidth = kMaxLblWidth;
     self.lblSpacing = kLblSpacing;
-    self.enableContinueLabel = kEnableContinueLabel;
-    self.enableSkipButton = kEnableSkipButton;
     
     // Shape layer mask
     mask = [CAShapeLayer layer];
     [mask setFillRule:kCAFillRuleEvenOdd];
     [mask setFillColor:[[UIColor colorWithHue:0.0f saturation:0.0f brightness:0.0f alpha:0.9f] CGColor]];
     [self.layer addSublayer:mask];
+    CGRect layerBounds = self.layer.bounds;
+    layerBounds.origin = CGPointMake(-kShadowLayerOffset, -kShadowLayerOffset);
+    CGSize layerSize = layerBounds.size;
+    layerSize.height += (2 * kShadowLayerOffset);
+    layerSize.width += (2 * kShadowLayerOffset);
+    layerBounds.size = layerSize;
+    self.layer.bounds = layerBounds;
+    self.layer.masksToBounds = NO;
+    self.layer.shadowColor = _maskColor.CGColor;
+    self.layer.shadowOffset = CGSizeMake(0.0, 0.0);
+    self.layer.shadowOpacity = 1.0;
+    self.layer.shadowRadius = kShadowLayerOffset;
```

Comment:
Drawing code is always a little opaque, so a little comment explaining what the intent of this stuff is for would be good.